### PR TITLE
Update ruby to 2.7.2

### LINKED ISSUE #198: Update version of Rails, Ruby and libraries
Break this into separate stories in whatever manner makes this work more manageable and approachable.

### PR DESCRIPTION
**Pull request by aterrype**
Update Ruby to the current version, and do updates and fixes that follow from that

Fixes #198 (since dependabot is keeping everything else up to date)

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@
 require:
   - rubocop-performance
   - rubocop-rails
+  - rubocop-rake
```
Comment (falonofthetower) [inline .rubocop.yml:7]: Out of curiosity what pushed you to include these along with 2.7.2?

Comment (aterrype) [reply .rubocop.yml:7]: When I ran rubocop after updating (to check for new cops, mostly), it gave me a suggestion to include those, which I hadn't seen before. I figured including them would both be a good idea in general and make the build cleaner, so I went ahead and did.

Comment (falonofthetower) [reply .rubocop.yml:7]: Oh, that's really neat. Love it.

**Review by falonofthetower — APPROVED** [review]
:shipit:

**Pull request merged**